### PR TITLE
SRCH-3996 videos page integration

### PR DESCRIPTION
**Pull request by yogesh27**
## Summary
- Videos page integration
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

**Review by stevenbarragan — DISMISSED** [review]
LGTM

File: app/javascript/test/Video.test.tsx [hunk 1]
```diff
@@ -2,18 +2,27 @@ import '@testing-library/jest-dom';
...
+  
   it('renders videos component', () => {
     render(
-      <Videos />
+      <Video {...video}/>
     );
   });
 
-  it('shows vidoe image and titlee', () => {
+  it('shows vidoe image and title', () => {
```
Comment (stevenbarragan) [inline app/javascript/test/Video.test.tsx:23]: typo `vidoe` => `video`

**Review by stevenbarragan — CHANGES_REQUESTED** [review]
Please fix typo

File: app/javascript/test/Video.test.tsx (hunk 1 above)
Comment (yogesh27) [reply app/javascript/test/Video.test.tsx]: this has been fixed

**Review by stevenbarragan — APPROVED** [review]

**Pull request merged**